src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -118,6 +118,7 @@ class OperatorStore extends BaseStore {
       selectOperators: action.bound,
       operatorCurrentFee: observable,
       unselectOperator: action.bound,
+      getMaxValidators: action.bound,
```

Comment:
Please rename it to `getOperatorValidatorsLimit`

Reply:
+